OneCommunityGlobal/HGNRest
# Vishala Check Types Modal

# Description
This PR implements a Modal to display the list of requested Types in the db such as Material , Equipment , Consumable , Reusable , Tool , and All. 
Implements # (WBS) 5.1.4

## Related PRS (if any):
To test this backend PR you need to checkout the [FE](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1802) frontend PR.

## Main changes explained:
- Added getTypesByType API

## How to test:
1. check into current branch
2. do `npm run build` and `npm run start` to run this PR locally
